content/background.html

```diff
@@ -92,21 +91,25 @@ <h3> International students </h3>
   <div id="international"></div>
 </div>
 
-<div class="vbx2"></div>
-
 <h3>Immigrated to Canada</h3>
-<p class="half">
+<p class="full">
   <span class="big">
   35.7% of students, or 39% of domestic students immigrated to Canada.
   </span>
-  Immigrated students came from countries including [TO ADD IF WE GET THIS INFO LATER]
+  Immigrated students are any students who are considered domestic students and <a href="https://uwaterloo.ca/graduate-studies-academic-calendar/general-information-and-regulations/fee-schedule-and-tuition-assessment-information" target="_blank">do not pay
+  as a result international tuition</a>, and among others are naturalized citizens or are Permanent Residents of Canada.
+  Immigrated students came from countries including Belgium, China, England and India.
 </p>
-<div class="half second">
+<div class="vb"></div>
+<div class="half">
   <span class="graph-title">Students' Immigrated to Canada</span>
```

Comment:
```suggestion
  <span class="graph-title">Students Immigrated to Canada</span>
```

though the "yes"/"no" don't quite match the title; can we go with something like a question on the students' immigration status? (or rename the yes/no to immigrated/non-immigrant)